### DISCUSSION TITLE
opening application

### DISCUSSION THREAD

**Discussion by thelaneburch** [ordinals/ord#1600] [Q&A] [answered]
I just downloaded the 0.4.2 zip file and am having trouble opening the application. I decompressed the file and am now trying to open the application. Could someone please help?

**Comment by raphjaph** [comment, marked as answer]
Did you follow the instructions here: https://docs.ordinals.com/guides/inscriptions.html?

**Comment by thelaneburch** [comment]
I opened and read instructions. Thanks. App install UI is something that needs work.

**Reply by raphjaph** [reply]
This is alpha stage software so everything is still very rough. You gotta be comfortable with the command line. Glad your making the effort to install though!

**Reply by thelaneburch** [reply]
I have bitcoin core downloaded. Can you help me with the next steps please?

**Reply by raphjaph** [reply]
Please visit the Discord for help: https://discord.gg/ordinals
